benchmark_runner/main/environment_variables.py

```diff
@@ -59,7 +76,9 @@ def __init__(self):
 
         # Choose default namespace
         base_workload = self._environment_variables_dict['workload'].split('_')[0]
-        if base_workload in self._environment_variables_dict['workload_namespaces']:
+        if os.environ.get('NAMESPACE'):
```

Comment:
Need to verify that we keep backward compatibility for other namespaces

Reply:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206